_data/simple-icons.json

```
@@ -3821,6 +3821,12 @@
             "hex": "000000",
             "source": "https://www.eyeem.com"
         },
+        {
+            "title": "F Sharp",
+            "hex": "378BBA",
+            "source": "https://foundation.fsharp.org/logo",
+            "guidelines": "https://foundation.fsharp.org/logo"
```

```suggestion
            "title": "F#",
            "slug": "fsharp",
            "hex": "378BBA",
            "source": "https://foundation.fsharp.org/logo",
            "guidelines": "https://foundation.fsharp.org/logo",
            "aliases": {
                "aka": [
                    "F Sharp"
                ]
            }
```

@LitoMore - how do you feel about the above? It's widely referred to as `F#` and `The F# Foundation` - so ideally we should display the title as such. Have added in a slug an alias just to cover our backs.

---

I had the same doubt!

---

Might be worth trying this change locally, and see if the linter falls out with you. What's the worst that could happen? 😅😐

---

`F#` looks good to me.

BTW, we have a logic for mapping special characters.

https://github.com/simple-icons/simple-icons/blob/da7cb03a790d86d3cef733095df9e60fb0de79ff/sdk.mjs#L15-L27

But I don't think we should update `TITLE_TO_SLUG_REPLACEMENTS`.

For `F# (fshrap)` or `C# (sharp)`, it's fine. But in the future universe, we may have an icon named `Simple Icons #1` or others. Its name should not be `simpleiconssharp1`.

So the extra `slug` field for this icon is reasonable.

---

@PratyushJayachandran - let's GO!
If you can make the change suggested above, we'll merge it in and see if anything breaks 🤞

---

Done